### PR TITLE
Magento 2.3.2 Release Note update for view model behavior

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request adds information about a change in the default view model behavior for v2.3.2. 

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4904) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4904) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ group: release-notes
 title: Magento Commerce 2.3.2 Release Notes
 ---
 
-*Patch code and release notes published on June 25, 2019. Release notes last updated on June 26, 2019.*
+*Patch code and release notes published on June 25, 2019. Release notes last updated on July 7, 2019.*
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]: It's the 8th now.

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ group: release-notes
 title: Magento Open Source 2.3.2 Release Notes
 ---
 
-*Patch code and release notes published on June 25, 2019. Release notes last updated on June 26, 2019.*
+*Patch code and release notes published on June 25, 2019. Release notes last updated o, 2019.*
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: "on" is misspelled, and the full date is missing.

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

**Pull request merged**